pom.xml

```diff
@@ -60,15 +199,6 @@
                     </compilerArgs>
                 </configuration>
             </plugin>
-            <plugin>
```

Comment:
Is this causing issues with Kotlin?

Reply:
Nope, this removal was unintentional.  Added it back.